README.md

```diff
@@ -6,12 +6,25 @@ Good candidates for things to include here:
 * Any library that is used by more than one or two PL projects
 * Any dependencies necessary to resolve transitive dependency conflicts between downstream PL projects
 
+## Contributing
+
+All updates should be made via pull request (PR).
+
+Updates should include a CHANGELOG entry and commit message with a brief explanation of what is being updated and why.
+
+#### Approving PRs
+In order to give meaningful approval to a PR, you must either run your component's test suite against the changes or confirm that your component does not use the dependencies being updated. **Please still approve the PR even if you aren't using the dependencies in question.**
+
+If your teammate opened the PR, you may assume that they ran the appropriate tests and you may approve as a sanity check.
```

Comment:
Not sure that sentence is needed, but if so, can we use something other than ‘sanity’? I’m kinda fond of ‘Spock’.

Reply:
ah yeah, thanks for calling that out! I will update.

I guess the reason I added this sentence was to imply that someone from the author's team should still look at the PR, but I can definitely make that clearer.

Reply:
updated!